src/main/java/org/dependencytrack/tasks/LockName.java

```diff
@@ -0,0 +1,6 @@
+package org.dependencytrack.tasks;
+
+public enum LockName {
+    PORTFOLIO_METRICS_TASK_LOCK,
+    PROJECT_METRICS_TASK_LOCK,
```

Comment:
will PROJECT_METRICS_TASK_LOCK be used later on ?

Reply:
created for Project metrics task which could be used later on. We can remove it later if we don't want to lock it